### PR TITLE
Loa 273 fix UI elements

### PR DESCRIPTION
**Pull request by Kostanix**

### CODE REVIEW

**Review by DorinaSobiecki — APPROVED** [review]
lgtm

File: app/src/main/res/layout/fragment_main_page.xml [hunk 1]
```diff
@@ -11,48 +11,52 @@
     The MapView is generated as the bottom object.
     So that the imageButton etc. are generated on top of the map, the MapView must be at the top of the xml file.
     -->
-    <com.mapbox.maps.MapView xmlns:mapbox="http://schemas.android.com/apk/res-auto"
+    <com.mapbox.maps.MapView
```
Comment (JohannesHausch) [inline app/src/main/res/layout/fragment_main_page.xml:14]: Why did you delete the link?

**Review by JohannesHausch — COMMENTED** [review]
Suggested changes are in the comments!
